db/dev_seeds.rb

```diff
@@ -872,6 +871,17 @@ def unique_document_number
   end
 end
 
+section "Geolocating Proposals"  do
```

Comment:
Investments are mappable as well, better to have a "Creating geolocations" section that has a loop that goes through both models without duplicated code